en/micro-integrator/docs/setup/installation/run_in_docker.md

```diff
@@ -40,6 +40,76 @@ Two docker images are available for the Micro Integrator:
     ```bash
     wso2/micro-integrator:1.0.0
     ```
+    
+## Build and run on Docker using Integration Studio
+
+Let’s create a simple proxy service using Integration studio and deploy it in a docker container. 
+
+For this we can use following synapse config
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy name="HelloWorld" startOnLoad="true" transports="http https" xmlns="http://ws.apache.org/ns/synapse">
+    <target>
+        <inSequence>
+            <payloadFactory media-type="json">
+                <format>{"Hello":"World"}</format>
+                <args/>
+            </payloadFactory>
+            <respond/>
+        </inSequence>
+        <outSequence/>
+        <faultSequence/>
+    </target>
+</proxy>
+```
+
+So now we can create the docker project with config following the steps 
+
+[Create Docker Project](../../develop/create-docker-project.md)
+
+If you put the 
```

Comment:
It is not clear where you put this Target Repository and Target Tag. Is it a configuration file that you are modifying here, or are they properties sent with the "docker image ls" command? If it is the latter, have the docker command in the exact format in which the user will need to issue it.

Suggestion:

"Build the Docker image with `sample proxy` as the target repository and `1.0.0` as the target tag. To do this, issue the following command:
<COMMAND>